### PR TITLE
docs: update README.md

### PR DESCRIPTION
**Pull request by li-jin-gou**
pr for https://github.com/cloudwego/kitex/issues/496

zh: 添加Etcd鉴权的示例代码
en: Example instructions for adding Etcd authentication

### CODE REVIEW

**Comment by li-jin-gou** [comment]
for 循环内出现 select-break 会导致 reviewdog 检查失败

File: README.md [hunk 1]
```diff
@@ -57,6 +57,72 @@ func main() {
...
+
+func (h *HelloImpl) Echo(ctx context.Context, req *api.Request) (resp *api.Response, err error) {
+	resp = &api.Response{
+		Message: req.Message,
+	}
+	return
+}
+
+func main() {
+	r, err := etcd.NewEtcdRegistryWithAuth([]string{"127.0.0.1:2379"}, "test", "test")
```
Comment (GuangmingLuo) [inline README.md]: 换成 username:password 呢？或者至少加个注释说明下

File: README.md [hunk 2]
```diff
@@ -57,6 +57,72 @@ func main() {
...
+package main
+
+import (
+    ...
+	"github.com/cloudwego/kitex/client"
+	etcd "github.com/kitex-contrib/registry-etcd"
+)
+
+func main() {
+	r, err := etcd.NewEtcdResolverWithAuth([]string{"127.0.0.1:2379"}, "test", "test")
```
Comment (GuangmingLuo) [inline README.md]: 同上

File: README.md (hunk 1 above)
Comment (li-jin-gou) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (li-jin-gou) [reply README.md]: done

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**